assets/js/components/PermissionsModal/AuthenticatedPermissionsModal.js

```diff
@@ -93,6 +96,15 @@ const AuthenticatedPermissionsModal = () => {
 		return null;
 	}
 
+	if (
+		unsatisfiedScopes &&
+		permissionsError?.data?.scopes.every( ( i ) =>
+			unsatisfiedScopes.includes( i )
```

Comment:
Can we rename it to `scope` for clarity?

```suggestion
		permissionsError?.data?.scopes.every( ( scope ) =>
			unsatisfiedScopes.includes( scope )
```